bin/close-pull-requests.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+
+COMMENT='Thank you for your pull request.
+However, you have submitted this PR on the Illuminate organization which is a read-only sub split of \`laravel/framework\`.
+Please submit your PR on the https://github.com/laravel/framework repository.
+
+Thanks!'
+
+for REPO in $(gh repo list illuminate --public --topic="read-only" --json "nameWithOwner" --jq '.[].nameWithOwner'); do
```

Comment:
`read-only` topic is just an example here, it could be something else

Reply:
What does this mean/do? Is a topic a label?

Reply:
Nvm got it.